src/SDDPoptimize.jl

```diff
@@ -208,42 +209,100 @@ function optimize(model::SPModel,
 
     stopping_test::Bool = false
     iteration_count::Int64 = 0
-    n::Int64 = param.forwardPassNumber
-
-
 
     while (iteration_count < param.maxItNumber) & (~stopping_test)
+        # Time execution of current pass:
+        tic()
+
         # Build given number of scenarios according to distribution
         # law specified in model.noises:
         aleas = simulate_scenarios(model.noises ,
                                     (model.stageNumber,
                                      param.forwardPassNumber,
                                      model.dimNoises))
+
+        # Forward pass
         costs, stockTrajectories, _ = forward_simulations(model,
                             param,
                             V,
                             problems,
-                            n,
                             aleas)
 
-        V0 = backward_pass(model,
+        # Backward pass
+        V0 = backward_pass!(model,
```

Comment:
I don't understand this V0. It should be a lower bound (called lb for example), obtained by the value of the polyhedral function V0^k at x0. I do not see it computed this way in backward_pass!